File: codeforces/17.word/word.py
# Author: José Rodolfo (jric2002)
s = str(input())
number_lowercase_letters = 0
number_uppercase_letters = 0
for position in range(0, len(s)):
  if (s[position].islower()):
    number_lowercase_letters += 1
  else:
    number_uppercase_letters += 1
if (number_lowercase_letters >= number_uppercase_letters):
  s = s.lower()
else:
  s = s.upper()
print(s)

File: codeforces/17.word/word.cpp
#include <iostream>
#include <cstring>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  string s;
  unsigned short int number_lowercase_letters = 0, number_uppercase_letters = 0;
  cin >> s;
  for (unsigned short int position = 0; position < s.size(); position++) {
    if (islower(s[position])) {
      number_lowercase_letters++;
    }
    else {
      number_uppercase_letters++;
    }
  }
  if (number_lowercase_letters >= number_uppercase_letters) {
    for (unsigned short int position = 0; position < s.size(); position++) {
      s[position] = tolower(s[position]);
    }
  }
  else {
    for (unsigned short int position = 0; position < s.size(); position++) {
      s[position] = toupper(s[position]);
    }
  }
  cout << s << endl;
  return 0;
}